cms/plugin_base.py

```diff
@@ -329,6 +329,9 @@ def requires_reload(self, action):
             reload_required = options.get('requires_reload', False)
         return reload_required
 
+    def get_use_cache(self, context, instance, placeholder):
+        return self.cache
```

Comment:
I'll rename the attribute as "use_cache" and add a property to handle the old setting